mscore/editstyle.ui

```diff
@@ -706,7 +748,7 @@
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <property name="verticalSpacing">
-            <number>-1</number>
+            <number>6</number>
```

Comment:
thiese -1 are used to keep the dialog as small as possible

Reply:
@AnandHemachandran will you apply the changes?